packages/nuxt/build.config.ts

```diff
@@ -2,6 +2,5 @@ import { defineBuildConfig } from 'unbuild';
 
 // Build Config for the Nuxt Module Builder: https://github.com/nuxt/module-builder
 export default defineBuildConfig({
-  // The devDependency "@sentry-internal/nitro-utils" triggers "Inlined implicit external", but it's not external
-  failOnWarn: false,
+  failOnWarn: false
```

Comment:
l: should we just remove this, and use the default? 
```suggestion
  failOnWarn: false
```

Reply:
...and there is a warning about the types 
`Potential missing package.json files: build/types/index.types.d.ts`

As far as I know there is no issue with that and this could be a warning cause by the 2-step build.